config-model/src/main/java/com/yahoo/searchdefinition/derived/RankProfileList.java

```diff
@@ -35,13 +35,15 @@
     private final RankingConstants rankingConstants;
     private final LargeRankExpressions largeRankExpressions;
     private final OnnxModels onnxModels;
+    private final boolean dryRunOnnxOnSetup;
 
     public static RankProfileList empty = new RankProfileList();
 
     private RankProfileList() {
         rankingConstants = new RankingConstants();
         largeRankExpressions = new LargeRankExpressions();
         onnxModels = new OnnxModels();
+        dryRunOnnxOnSetup = true;
```

Comment:
This seems a bit eager?

Reply:
This is just test code. fail-fast is my preferred apparoach.